cmd/main.go

```diff
@@ -82,7 +82,7 @@ func main() {
 	}
 
 	comMatrixFileName := filepath.Join(destDir, fmt.Sprintf("communication-matrix.%s", format))
-	err = os.WriteFile(comMatrixFileName, []byte(string(res)), 0644)
```

Comment:
This is done in two more places in this file

Reply:
done, found 1 more occasion of this